serverFiles/mailer.js

```diff
@@ -0,0 +1,56 @@
+var nodemailer = require('nodemailer');
+
+var transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: 'YOUR_EMAIL_HERE@gmail.com',
+      pass: 'YOUR_PASSWORD_HERE'
+    }
+  });
+
+  var sendMail = (userInfo,callback) => {
+
+    console.log(userInfo);
+    var send = `Hi ${userInfo.name}, \n
+  We recieved your Feedback :),
+  Thanks for the feedback. :)
+  Hope to see you soon.\n
+  REGARDS,
+  TEAM DevelopersRising`;
+  
+    var sendSelf = `Feedback from ${userInfo.name}, \n
+  ${userInfo.feedback}`;
+  
+  
+    var mailOptions = {
```

Comment:
Since you're using ES6, you can replace the use of var within function scope to let.